spamdb/modules/clas.py

```diff
@@ -0,0 +1,94 @@
+import random
+from string import ascii_letters, digits
+from modules.env import env
+import modules.util as util
+from modules.user import User
+
+def update_clas_colls() -> None:
+    args = env.args
+    db = env.db
+
+    if args.drop:
+        db.clas.drop()
```

Comment:
Are these the right collection names? they don't match the ones in bulk_write below

Reply:
you are completely right